swagger/swagger.json

```diff
@@ -727,6 +727,29 @@
           "example" : "Joe.Bogs@example.org",
           "description" : "email",
           "readOnly" : true
+        },
+        "return_url" : {
+          "type" : "string",
+          "example" : "https://service-name.gov.uk/transactions/12345",
+          "description" : "service return url",
+          "readOnly" : true,
+          "minLength" : 0,
+          "maxLength" : 2000
+        },
+        "delayed_capture" : {
+          "type" : "boolean",
+          "example" : false,
+          "description" : "delayed capture flag",
+          "readOnly" : true
+        },
+        "metadata" : {
+          "readOnly" : true,
+          "$ref" : "#/definitions/ExternalMetadata"
+        },
+        "prefilled_cardholder_details" : {
+          "description" : "prefilled_cardholder_details",
+          "readOnly" : true,
+          "$ref" : "#/definitions/PrefilledCardholderDetails"
```

Comment:
Why is this stuff appearing now?

Reply:
argg keeps trying to change swagger.json, don't know why. I'll remove from this PR.